packages/frontend-2/components/settings/user/profile/DeleteAccountDialog.vue

```diff
@@ -68,12 +64,15 @@ const stringMatchesEmail: GenericValidateFunction<string> = (val: string) => {
 
 const emit = defineEmits<{
   (e: 'deleted'): void
+  (e: 'update:open', val: boolean): void
 }>()
 
 const props = defineProps<{
   user: UserProfileEditDialogDeleteAccount_UserFragment
 }>()
 
+const isOpen = defineModel<boolean>('open', { required: true })
```

Comment:
use `defineModel`

Reply:
> use `defineModel`

Not sure what you mean, it's already `defineModel` here? I'll check if there any others that are computed and update them

Reply:
Ah my bad, I was confused cause you also added the `update:open` event definition - that's redundant as that's what `defineModel` does already